app/install/google-cloud.md

```diff
@@ -0,0 +1,104 @@
+---
+id: page-install-method gke-deployment
+title: Install - Google Cloud Platform
+header_title: Google Cloud Platform
+header_icon: /assets/images/icons/icn-installation.svg
+breadcrumbs:
+  Installation: /install
+links:
+  gke_repo: "https://github.com/Mashape/kong-distribution-gke"
+  gcloud_sdk: "https://cloud.google.com/sdk/"
+  kubectl_sdk: "https://cloud.google.com/container-engine/docs/quickstart#install_the_gcloud_command-line_interface"
+---
+
+Kong can easily be povisioned to Google Cloud using the following templates:
+
+- [Download GKE deployment templates]({{ page.links.gke_repo }})
+- 
+
+----
+
+### Notes:
+
+1. **Initial setup**:
+
+    You would need [gcloud]({{ page.links.gcloud_sdk }}) and [kubectl]({{ page.links.kubectl_sdk }}) command-line tools installed and set up to run deployment commands.
+
+    Download or clone the aforementioned repo
+
+    ```bash
+    $ git clone git@github.com:Mashape/kong-dist-google-cloud.git
+    $ cd kong-dist-google-cloud
+    ```
+
+2. **Deploy a cluster**:
+
+    Using `cluster.yaml`, deploy a GKE cluster to use for deploying the solution later. Fill in the following information before deploying:
```

Comment:
I think both 1 and 2 change the figure of speech. Here user suppose to provision cluster 1st and later all the kong resources in it. May be  instead of 'deploying the solution', I should use 'deploying Kong resources'